subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -345,7 +345,18 @@ int lll_done(void *param)
 #endif /* !CONFIG_BT_CTLR_LOW_LAT_ULL_DONE */
 
 #if defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
-	lll_done_score(param, 0, 0); /* TODO */
+	struct event_done_extra *extra;
+	uint8_t result;
+
+	/* TODO: Pass from calling function */
+	result = DONE_COMPLETED;
+
+	lll_done_score(param, result);
+
+	extra = ull_event_done_extra_get();
+	LL_ASSERT(extra);
+
+	extra->result = result;
```

Comment:
Add a note, why `extra->type`  is unassigned?

Reply:
Type is set by the role prior to lll_done. Adding comment to code.